src/pages/workspaces/workspace/Data.js

```diff
@@ -557,21 +557,29 @@ const WorkspaceData = _.flow(
                 ])],
                 () => div({ role: 'list', style: { fontSize: 14, lineHeight: '1.5' } }, [
                   _.map(([tableName, { count }]) => {
+                    const canCompute = !!(workspace?.canCompute)
                     return h(DataTypeButton, {
-                      buttonStyle: { borderBottom: 0, height: 40 },
+                      buttonStyle: canCompute ? { borderBottom: 0, height: 40 } : { borderBottom: 0, height: 40, color: colors.dark(0.25) },
+                      tooltip: canCompute ?
+                        tableName ? `${tableName} (${count} row${count === 1 ? '' : 's'})` : undefined :
+                        [div({ key: `${tableName}-tooltip`, style: { whiteSpace: 'pre-wrap' } }, 'You must be an owner, or a writer with compute permission, to view this snapshot.\n\n' +
+                        'Contact the owner of this workspace in order to change your permissions.')],
```

Comment:
Someone once told me to only use "in order" when putting things in order. Your call.
```suggestion
                        'Contact the owner of this workspace to change your permissions.')],
```

Reply:
heh, that's the verbiage provided from the UX team … @jeromechadel do you have an opinion here?